cockroachcloud/cloud-api.md

```diff
@@ -40,6 +40,37 @@ Authorization: Bearer {secret_key}
 
 Where `{secret_key}` is the [secret key string you stored when you created the API key in the Console](console-access-management.html#create-api-keys).
 
+## Set the API version
+
+The `Cc-Version` HTTP header specifies the version of the Cloud API to use. If you omit the `Cc-Version` header, the Cloud API will default to version `2022-03-31` (the initial release of the Cloud API). The Cloud API uses date-based versions of the form `YYYY-MM-DD`, in [ISO 8601 format](https://www.w3.org/TR/NOTE-datetime).
```

Comment:
@marksoper Should we mention here that this behavior is temporary? I think we might want this message to be from the perspective that you should always be using the `cc-version` header, but for backwards compatibility reasons, we will support omitting it for a while (I think we agreed on 3 months?).

Maybe something like this:
> The `Cc-Version` HTTP header specifies the version of the Cloud API to use. The Cloud API uses date-based versions of the form `YYYY-MM-DD`, in [ISO 8601 format](https://www.w3.org/TR/NOTE-datetime).
>
> Note: Until January 1, 2023, omitting the `Cc-Version` header will default to version `2022-03-31` (the initial release of the Cloud API). After January 1, 2023, omitting the `Cc-Version` header will return an HTTP 400 response with the message "Missing Cc-Version".

Reply:
In general, we avoid documenting potential future changes in the docs. The docs describe how the product works at the present. If we decide to change how we implement version pinning, we've already set expectations among our customers, and caused unnecessary confusion.

Reply:
If we implemented a deprecation warning when users omit the version header, we would add a corresponding warning in the docs.

Reply:
If that's a hard line, then I would just leave out the part about what happens when you omit the version header. At the time that these docs are going up, the official stance is that you need to include the version header. We're going to have a default temporarily for backwards compatibility reasons, but IMO that's something that only existing users should know about.